Data contribution - Module 1 (#36)
arcus/Arcus_Labs_Orientation: new_data_contribution.md

If after viewing this module, you are prepared to archive data with Arcus, please fill out [the following request](https://pm.arcus.chop.edu/servicedesk/customer/portal/6/create/256) to start the process.

### Arcus Lab users who have data to contribute  

When appropriate, archiving your research in Arcus is expected with a Scientific Project with an Arcus Lab. This is documented in the [Arcus Terms of Use](https://arcus.chop.edu/terms-of-use). Archiving is required if you would like to move any data created within an Arcus Lab to a new Scientific Project with an Arcus Lab or if other research teams would like to re-use your data.

When you are ready to archive your lab data, please submit the following request in the [Arcus Help Center](https://pm.arcus.chop.edu/servicedesk/customer/portal/6/create/256) to begin the data contribution process.

> During the orientation for your Arcus Lab, you should have received an orientation to the Project Template structure. A review of this information is available in Module 2 [ADD LINK LATER] and in this [Arcus forum post](https://forum.arcus.chop.edu/t/the-arcus-project-template/255). Prior to archiving, the research data and all associated reference information needs to be organized in the project template directory preloaded into your lab. If you need assistance in arranging your data, please request a consultation by the Arcus Library Sciences team using the [following link](https://jira.arcus.chop.edu:8443/servicedesk/customer/portal/6/create/278?q=rdm&q_time=1690816635148).

Reviewer: Just a reminder to fix before final approval

Author: We took your suggestion below, thanks for catching and reminding us to clean up this note.

> During the orientation for your Arcus Lab, you should have received an orientation to the Project Template structure. A review of this information is available in Module 2 [ADD LINK LATER] and in this [Arcus forum post](https://forum.arcus.chop.edu/t/the-arcus-project-template/255). Prior to archiving, the research data and all associated reference information needs to be organized in the project template directory preloaded into your lab. If you need assistance in arranging your data, please request a consultation by the Arcus Library Sciences team using the [following link](https://jira.arcus.chop.edu:8443/servicedesk/customer/portal/6/create/278?q=rdm&q_time=1690816635148).

Reviewer: ```suggestion
During the orientation for your Arcus Lab, you should have received an orientation to the Project Template structure. A review of this information will be arriving soon in a second training module here, and is also available  in this [Arcus forum post](https://forum.arcus.chop.edu/t/the-arcus-project-template/255). Prior to archiving, the research data and all associated reference information needs to be organized in the project template directory preloaded into your lab. If you need assistance in arranging your data, please request a consultation by the Arcus Library Sciences team using the [following link](https://jira.arcus.chop.edu:8443/servicedesk/customer/portal/6/create/278?q=rdm&q_time=1690816635148).
```
Does this work for now for you?

### Data Reuse

**Collate data in Arcus for use in lab**

As mentioned, contributing research data is prerequisite to working with that data in an Arcus lab. Once data is stored in Arcus it can be both integrated with a variety of data sources and successfully delivered to a lab. Our privacy team has regulatory language that allows for expansive and evolving integration. Arcus labs are available at a persistent web URL that facilitates collaborative research across multiple users.

**Open Science initiatives**

Broadly speaking, Open Science Initiatives promote making publicly funded research data more widely available and collecting and organizing these data in a way that makes them reusable. For example, as of 2023, all NIH funded studies regardless of size must develop a [data management and sharing plan](https://www.research.chop.edu/sites/default/files/2022-11/DBHi_Converting_Resource_DMSP.pdf) which stipulates data is ultimately deposited in a searchable data repository. Archiving your data in Arcus ensures that your data is in compliance with this requirement.

### Grant Requirements

**FAIR Data**

> ![FAIR data principles graphic](media/FAIR_data_principles.png)

Reviewer: ```suggestion
![FAIR data principles graphic](media/FAIR_data_principles.png)<!-- style = "max-width: 600px" -->
```
I think it looks better slightly smaller but maybe that's just me?  Take it or leave it!

<div class = "learn-more">
<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>

For more information about requesting and using an Arcus Lab, please see the [Arcus Lab Orientation module](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/main/arcus_orientation.md#1).

</div>

## Pre-Data Contribution 

> ![Graphic of a server with a plus sign](media/data_contribution_server.png)

Reviewer: ```suggestion
![Graphic of a server with a plus sign](media/data_contribution_server.png)<!-- style = "max-width: 150px;" -->
```
Again, feel free to veto, I just think it looks better a bit smaller.

<!--

author:   Arcus Library Sciences
email: dlarcuslibraryscience@chop.edu
version:  1.0.0
current_version_description: Brief description of why this version exists
module_type: standard
docs_version: 1.1.0
language: en
narrator: UK English Female
title: Arcus Data Contribution Orientation
comment:  Learn about archiving research data with Arcus
long_description: Arcus Archives is the canonical repository for research data at CHOP. Archiving research data preserves the important research performed at CHOP according to archival standards while facilitating data sharing. This module reviews reasons to archive research data at Arcus, and the data scoping, privacy review and technical considerations taken before receiving data. Module 2 covers the steps of archiving the data at Arcus.

estimated_time: 30 minutes

> @pre_reqs: It's helpful to have reviewed the [Arcus website](https://arcus.chop.edu) at  (available only on the CHOP network), to understand Arcus’s overall goals.

Reviewer 2: ```suggestion
@pre_reqs
It's helpful to have reviewed the [Arcus website](https://arcus.chop.edu) at  (available only on the CHOP network), to understand Arcus’s overall goals.
```

This is what's called a "block macro" in liascript, so the text goes between the @pre_reqs and @end tags rather than after a colon. The lone @end after this macro was causing the error that then prevented the next macro (learning_objectives) from displaying properly.
